Doc/howto/logging-cookbook.rst

```diff
@@ -333,7 +333,7 @@ Suppose you configure logging with the following JSON:
     }
 
 This configuration does *almost* what we want, except that ``sys.stdout`` would
-show messages of severity ``ERROR`` and above as well as ``INFO`` and
+show messages of severity ``ERROR`` and only events of this level and higher severity will be tracked as well as ``INFO`` and
```

Comment:
Would prefer the wording "only events of this severity and higher will  be tracked" here.